bedrock/lightbeam/templates/lightbeam/about.html

```diff
@@ -45,10 +45,14 @@ <h1 class="page-slogan">{{_('About Lightbeam')}}</h1>
       {{_('Lightbeam visualizes the relationships between the sites you visit and the third party sites that are active on those pages.')}}
     </p>
     <p>
+{% if l10n_has_tag('update_may2015) %}
```

Comment:
You're missing a closing `'` here in the tag name. It should be `{% if l10n_has_tag('update_may2015') %}`

While you're updating, if you could also indent these tags to sit inside the `<p>` that would be great. Thanks